documentation/bin/createDocu.groovy

```diff
@@ -302,7 +302,7 @@ class Helper {
 
         params.put('STEP_CONFIG', script.STEP_CONFIG_KEYS ?: [])
         params.put('GENERAL_CONFIG', script.GENERAL_CONFIG_KEYS ?: [] )
-        params.put('STAGE_CONFIG', script.PARAMETER_KEYS ?: [] )
+        params.put('STAGE_CONFIG', script.STEP_CONFIG_KEYS ?: [] )
```

Comment:
What about this:
```suggestion
        params.put('STAGE_CONFIG', script.STAGE_CONFIG_KEYS ?: script.STEP_CONFIG_KEYS ?: [] )
```

Reply:
I mentioned this fallback in the meeting, but it makes only sense if we keep two separate columns in the documentation. Last decision was to have one Stage/Step column. Should we change that again?